ui/src/authors/containers/CitationSummaryContainer.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Map } from 'immutable';
+
+import { fetchCitationSummary } from '../../actions/citations';
+import CitationSummaryTable from '../components/CitationSummaryTable/CitationSummaryTable';
+import LoadingOrChildren from '../../common/components/LoadingOrChildren';
+import ErrorAlertOrChildren from '../../common/components/ErrorAlertOrChildren';
+import { ErrorPropType } from '../../common/propTypes';
+
+class CitationSummaryContainer extends Component {
+  componentDidMount() {
+    this.fetchCitationSummary();
+  }
+
+  componentDidUpdate(prevProps) {
+    const prevSearchQuery = prevProps.searchQuery;
+    const { searchQuery } = this.props;
+    if (searchQuery !== prevSearchQuery) {
+      this.fetchCitationSummary();
+    }
+  }
+
+  fetchCitationSummary() {
+    const { searchQuery, dispatch } = this.props;
+    dispatch(fetchCitationSummary(searchQuery));
+  }
+
+  render() {
+    const {
+      citationSummary,
+      loadingCitationSummary,
+      error,
+      searchQuery,
+    } = this.props;
+    return (
+      <LoadingOrChildren loading={loadingCitationSummary}>
+        <ErrorAlertOrChildren error={error}>
+          {citationSummary && (
+            <CitationSummaryTable
+              citationSummary={citationSummary}
+              searchQuery={searchQuery}
+            />
+          )}
+        </ErrorAlertOrChildren>
+      </LoadingOrChildren>
+    );
+  }
+}
+
+CitationSummaryContainer.propTypes = {
+  loadingCitationSummary: PropTypes.bool.isRequired,
+  dispatch: PropTypes.func.isRequired,
+  searchQuery: PropTypes.instanceOf(Map).isRequired,
+  error: ErrorPropType, // eslint-disable-line react/require-default-props
+  citationSummary: PropTypes.instanceOf(Map),
+};
+
+CitationSummaryContainer.defaultProps = {
+  citationSummary: null,
```

Comment:
add `eslint-disable-line react/require-default-props` for it instead, since we have the default from `store`